src/main/java/edu/greenblitz/robotName/commands/swerve/MoveToClimbPosition.java

```diff
@@ -2,10 +2,12 @@
 
 import edu.greenblitz.robotName.Field;
 import edu.greenblitz.robotName.commands.auto.MoveToPosition;
+import edu.greenblitz.robotName.utils.FMSUtils;
+import edu.wpi.first.wpilibj.DriverStation;
 
 public class MoveToClimbPosition extends MoveToPosition {
 
     public MoveToClimbPosition() {
-        super(Field.ScoringPositions.CLIMB_POSITION);
+        super(FMSUtils.getAlliance() == DriverStation.Alliance.Blue ? Field.ScoringPositions.RED_CLIMB_POSITION : Field.ScoringPositions.BLUE_CLIMB_POSITION);
```

Comment:
If we are the blue alliance, why do we use the red climb position?
Also, I believe there is a function for isRedAlliance in FMSUtils

Reply:
> I believe there is a function for isRedAlliance in FMSUtils
![download](https://github.com/GreenBlitz/CRRRRESCENDO/assets/94316058/9fc161d2-110a-4866-81c8-828abbb1679e)